src/components/common/PlotlyCard/Component.js

```diff
@@ -8,77 +8,265 @@ import {
   CloudOutlined,
   HomeOutlined,
   TeamOutlined,
-  DollarCircleOutlined,
+  NodeIndexOutlined,
   DownOutlined,
 } from '@ant-design/icons';
 
 import { Switch } from 'antd';
 
```

Comment:
Not sure if this was just an overall name change or if you're getting this data from somewhere else now. id add a comment regarding what the change was

Reply:
We swapped one of the icons being used from the Ant Design library because we changed the overall category of information being displayed. Needed an icon that was more representative of Walkscore.